### ISSUE TITLE
Segmenting issue with whole 2D Image and different output in desktop, website, and code based application

### ISSUE THREAD

**Issue by FridaPallapies** [MouseLand/cellpose#460]
Hi, 

I’ve tried running cellpose on one of my images (1024x1024 pixel), but it fails to segment all the nuclei in the given image. But when I crop the Image down to 500x500 pixel it works perfectly fine. Problem with that is, that I can not crop down every image without losing some fluorescent signal. Is there any way to use the whole image? 

Here is the image I was using:

![stack_3_CTRL_DAPI_DG1_A_01](https://user-images.githubusercontent.com/77928601/156368177-ad4f26b5-3348-437f-9a40-a59cdfde8b8f.png)

I also tried out segmenting the same image first via the website, then in the desktop client and lastly through a direct integration of your source code in a python script on the same machine. And although I always ran it with the default settings for flow threshold etc. the output of the different versions seemed to differ, which makes it hard to validate the output.

I plotted the different outcomes depending on the tool I used and the image size: 

<img width="520" alt="image" src="https://user-images.githubusercontent.com/77928601/156350532-ef22c186-17ba-485a-938b-158d61321720.png">

Any help regarding those problems would be much appreciated. 

Thank you, 

Frida

**Comment by carsen-stringer** [comment]
sorry Frida it's hard to know why there would be a difference in the desktop version vs the code unless by desktop version you mean the old .exe file? also it would be helpful to see your commands in desktop vs code, did you set the diameter the same in the GUI vs the code?

**Comment by FridaPallapies** [comment]
Hi, thank you for your response. I did use a local installation instead of the old .exe file. Usually I didn't change any of the parameters except for the cell diameter, in which case I made sure that the values would match in both the GUI and the code based approach. 

Anyhow, the faulty segmentation mostly got resolved after reading this issue [https://github.com/MouseLand/cellpose/issues/454](url) and changing the values of the `normalize99` function.
